tests/area/test_area.py

```diff
@@ -46,88 +62,72 @@ def setup_method(self):
-        self.area.children = [self.area_child]
-        self.area.grid_fee_percentage = 1
-        self.dispatcher = AreaDispatcher(self.area)
-        self.stats = AreaStats(self.area._markets, self.area)
-
-    def teardown_method(self):
+    @staticmethod
+    def teardown_method():
         ConstSettings.IAASettings.MARKET_TYPE = SpotMarketTypeEnum.ONE_SIDED.value
         ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = BidOfferMatchAlgoEnum.PAY_AS_BID.value
         ConstSettings.GeneralSettings.EVENT_DISPATCHING_VIA_REDIS = False
         constants.RETAIN_PAST_MARKET_STRATEGIES_STATE = False
 
-    def test_respective_area_grid_fee_is_applied(self):
-        self.config.grid_fee_type = 2
-        self.area = Area(name="Street", children=[Area(name="House")],
-                         grid_fee_percentage=5, config=self.config)
-        self.area.parent = Area(name="GRID", config=self.config)
-        self.area.activate()
-        assert self.area.spot_market.fee_class.grid_fee_rate == 0.05
-        self.area.spot_market.offer(1, 1, "test", "test")
-        assert list(self.area.spot_market.offers.values())[0].price == 1.05
-
-    def test_delete_past_markets_instead_of_last(self):
+    @staticmethod
+    def test_respective_area_grid_fee_is_applied(config):
+        config.grid_fee_type = 2
+        area = Area(name="Street", children=[Area(name="House")],
+                         grid_fee_percentage=5, config=config)
+        area.parent = Area(name="GRID", config=config)
+        area.activate()
+        assert area.spot_market.fee_class.grid_fee_rate == 0.05
+        area.spot_market.offer(1, 1, "test", "test")
+        assert list(area.spot_market.offers.values())[0].price == 1.05
+
+    @staticmethod
+    def test_delete_past_markets_instead_of_last(config):
         constants.RETAIN_PAST_MARKET_STRATEGIES_STATE = False
-        self.area = Area(name="Street", children=[Area(name="House")],
-                         config=self.config, grid_fee_percentage=5)
-        self.area.activate()
-        self.area._bc = None
-
-        self.area.cycle_markets(False, False, False)
-        assert len(self.area.past_markets) == 0
-
-        current_time = today(tz=constants.TIME_ZONE).add(minutes=self.config.slot_length.minutes)
-        self.area._markets.rotate_markets(current_time)
-        assert len(self.area.past_markets) == 1
-
-        self.area._markets.create_new_spot_market(
-            current_time, AvailableMarketTypes.SPOT, self.area)
-        current_time = today(tz=constants.TIME_ZONE).add(minutes=2*self.config.slot_length.minutes)
-        self.area._markets.rotate_markets(current_time)
-        assert len(self.area.past_markets) == 1
-        assert (list(self.area.past_markets)[-1].time_slot ==
-                today(tz=constants.TIME_ZONE).add(minutes=self.config.slot_length.minutes))
-
-    def test_keep_past_markets(self):
+        area = Area(name="Street", children=[Area(name="House")],
```

Comment:
A general question: Internally in the code we will use `MarketBase` and `Area` exchangeably right?

Reply:
I am asking because of the serialising of configurations from the gsy-web side:
https://github.com/gridsingularity/gsy-e/blob/03f8ea4240b8b6678394be787b4a18c1b2a1b130/src/gsy_e/gsy_e_core/area_serializer.py#L136
Where we could also make the code a bit more explicit. 
IMO, no need to change all tests like here though.

Reply:
Can you please either adapt the `area_serializer.py` or explain me, why it does make sense? Thanks!

Reply:
Ok, I see. But we also have only-backend-users, too. Do I get it right: They will use the `Market` class from now on (as the new wiki will show) and old user-setups will work because we still support `Area`, right? 
The only point were I predict confusion is when the users look into our default/test setup files which are not in sync with the new wiki. But I guess  our costumer service can explain then :-)

Reply:
@hannesdiedrich Yes, this was discussed on slack a few days ago. Actually, that conversation didn't conclude and I worked on the other changes in the meanwhile. 
I'd personally update the existing setups with the new syntax. If you want me to amend all existing setups I can work on it (maybe in a new PR since there are lots of setups).

Reply:
This probably needs to be decided in the discussion. I can leave my 5 cents to it if you link me to it (privately in slack). Thanks!

Reply:
To take this discussion from the top and clarify the most important thing first: old Market -> MarketBase , new Market -> class that inherits from Area and can be used by the setup files for rebranding purposes, which inherits from Area. These are 2 totally distinct classes and in no way can they be used interchangeably. The latter could be told for new Market and Area (since the former inherits from the latter).
For the rest of the discussion, I personally do not mind to rename all setup files from Area to Market / Asset classes, however I find it a bit futile since the researchers have their own scripts and do not use our integration tests ones IIRC (for instance, see setup/api_setup or setup/myco_setup)